<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IntNumber</class>
 <widget class="QWidget" name="IntNumber">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>813</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="backIntNum">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>791</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Целые числа</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Первое число:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Второе число:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="firstInt">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>331</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="secondInt">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>50</y>
      <width>331</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="push_ABS_Z_N">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>160</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ABS_Z_N</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_POZ_Z_D">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>160</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>POZ_Z_D</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_MUL_ZM_Z">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>160</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>MUL_ZM_Z</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_TRANS_N_Z">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>160</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TRANS_N_Z</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_TRANS_Z_N">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>160</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TRANS_Z_N</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_ADD_ZZ_Z">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>210</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ADD_ZZ_Z</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_SUB_ZZ_Z">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>210</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>SUB_ZZ_Z</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_MUL_ZZ_Z">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>210</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>MUL_ZZ_Z</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_DIV_ZZ_Z">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>210</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>DIV_ZZ_Z</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_MOD_ZZ_Z">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>210</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>MOD_ZZ_Z</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>320</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ответ:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="answerInt">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>320</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="faqI">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>10</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>?</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
